src/pages/ProfileView.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { FaDiscord, FaEnvelope, FaGithubSquare, FaLink, FaLinkedin, FaMapMarkerAlt, FaPencilAlt } from 'react-icons/fa';
+import {
+    BIOGRAPHY,
+    DISCORED_USERNAME,
+    EMAIL,
+    FIRST_NAME,
+    GITHUB_USERNAME,
+    LAST_NAME,
+    LINKS,
+    LOCATION,
+    PROFILE_IMAGE,
+    SOCIAL,
+    USERNAME
+} from '../constants/user-constants';
+import IconPlaceholder from '../images/insert-profile-picture-here.png';
+import { useUser } from '../contexts/UserContext';
+
+export default function ProfileView() {
+    const [user] = useUser();
+
+    return (
+        <div className="profile-view-body">
+            {user && <>
+                <div className="user-info">
+                    {user[SOCIAL][PROFILE_IMAGE] ?
```

Comment:
Do we have a list of required data when a user is signing up? If not, you'll need to handle the case where the user hasn't provided certain attributes in the `Social` object. Hello optional chaining!

I like that you're using bracket notation and that the source of truth for all the user properties are in one place.

Reply:
only thing absolutely required are email and password :)

Reply:
Yep I wasn't sure what to do with some of the fields that aren't required, and if we want placeholders for anything like username, full name and bio. so I just setup a generic page, assuming the UX style changes will come later when the look is more defined.